src/index.js

```diff
@@ -27,7 +27,7 @@ if (process.env.NODE_ENV === 'development') {
 }
 
 if (process.env.NODE_ENV === 'production') {
-  const runtime = require('offline-plugin/runtime');
+  const runtime = require('@mstange/offline-plugin/runtime');
```

Comment:
Please add a comment with a link to https://github.com/NekR/offline-plugin/pull/410 and a comment about why this is needed.